doc/api/assert.md

```diff
@@ -330,6 +330,11 @@ changes:
 Asserts that the function `block` does not throw an error. See
 [`assert.throws()`][] for more details.
 
+Please note: Using `assert.doesNotThrow()` is not recommended because there is
+no benefit by catching an error and then rethrowing it. Instead, consider adding
+a comment next to the specific code path that should not throw and keep all
+error messages in your code as expressive as possible.
```

Comment:
Nit: `your` :-(

Perhaps

```
Instead, consider adding a comment next to the specific code path that should
not throw and keep add error messages as expressive as possible.
```

Reply:
@jasnell There's an error in your proposed text. I *think* the word `add` should be deleted?